csrankings.csv

```diff
@@ -17452,6 +17453,7 @@ Siddharth Srivastava,Arizona State University,http://siddharthsrivastava.net,H3Z
 Siddhartha Banerjee,Cornell University,https://people.orie.cornell.edu/sbanerjee,_kqpoHIAAAAJ
 Siddhartha Chaudhuri,IIT Bombay,https://www.cse.iitb.ac.in/~sidch,QMc-grEAAAAJ
 Siddhartha Sarma,IIT Mandi,http://faculty.iitmandi.ac.in/~siddhartha/index.html,c-oFx5UAAAAJ
+Siddhartha S. Srinivasa,University of Washington,https://goodrobot.ai/,RCi98EAAAAAJ

```

Comment:
Same issue: major leadership role, this time at Amazon. Likely not eligible for inclusion.